Capstone Project 1 : Chess Event Site

IndieCoderMM opened the pull request:
# Chess event website
For this capstone project, I am building a website for a chess event.
In this branch, I:
- Built mobile-first design for *home-page* and *about-page*
- Modified for the desktop version
- Added mobile menu
- Loaded contents in *feature* section dynamically from JavaScript

Here you can watch my [presentation about this project](https://www.loom.com/share/f776c2ee38fe487b89b186001d697429).

IndieCoderMM committed:
Create homepage for mobile

IndieCoderMM committed:
Add about page

IndieCoderMM committed:
Add navbar for desktop

IndieCoderMM committed:
Complete desktop version

IndieCoderMM committed:
Add mobile menu feature

IndieCoderMM committed:
Add toolbar to mobile menu

IndieCoderMM committed:
Load feature section with JS

IndieCoderMM committed:
Modify page design

IndieCoderMM committed:
Install linters

IndieCoderMM committed:
Fix linter errors

IndieCoderMM committed:
Add gradient title

IndieCoderMM committed:
Update documentation

mukhammadrizooff commented on style.css:
```diff
@@ -0,0 +1,553 @@
+  background-color: #16171bce;
+}
+
+.program-card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.25rem;
+  padding: 1rem;
+  background-color: rgba(51, 51, 51, 0.879);
+  z-index: 5;
+}
```
- [ ] [REQUIRED] Good job so far 👍 Please add `hover` effect for cards like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 

### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/203613058-b7a20b41-842f-41f6-bb23-7d2dfe376219.png)

### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

![image](https://user-images.githubusercontent.com/63915024/203613014-e4e0c2f7-b09d-4e20-ae5c-48a2fed72a0c.png)

mukhammadrizooff commented on main.js:
```diff
@@ -0,0 +1,95 @@
+    bio: 'Fabiano, an Italian-American grandmaster, is a chess prodigy who became a grandmaster at age 14.',
+    profile: 'img/fabiano.jpg',
+  },
+];
```
- [ ] [REQUIRED]  Good job so far! You have done a great job in this section, I would request you to make the following changes to this section. Please, the elements in this section, i.e. the number of players should be `6` but you have `4`, and the `title` part's the bottom and the `bio` part at the top should be a `small line` as given in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 At least I think it's easy for a smart student like you to do it 💪 

### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/203611823-dc7530eb-7883-461f-9455-e5ca02d66738.png)

### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

![image](https://user-images.githubusercontent.com/63915024/203611864-ba5f4bac-ac27-4063-bc30-54b14d5f163d.png)

mukhammadrizooff reviewed (changes requested):
Hi @IndieCoderMM, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

<img src="https://media4.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive summary ✔️ 
- Descriptive Presentation video ✔️ 
- Perfect mobile version design ✔️ 

## Suggested changes
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] In my opinion, the `width` and `height` of the **Header** section in the desktop version is good, but I think that the `height` value of the other `sections` is low, because the rest of the sections are reflected flat. It would be great if you could also implement this segestion 😃 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

IndieCoderMM committed:
Make requested changes

- Add two more items to feature section
- Redesign the feature grid
- Add divider to player's info
- Add hover effects
- Increase section height for desktop

emmiiorji reviewed (approved):
Hi @IndieCoderMM, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![congratulations-2805](https://user-images.githubusercontent.com/20554583/203687220-ff1a910f-f657-4970-a6e8-0c6fb31dbc7f.gif)

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- You have implemented requested changes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
